### PR TITLE
Move Orchard bundle commitments to the `orchard` crate.

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

**Review by daira — APPROVED** [review]
utACK (also compared against https://github.com/zcash/orchard/pull/140).

File: Cargo.toml [hunk 1]
```diff
@@ -24,5 +24,5 @@ nom = { git = "https://github.com/myrrlyn/nom.git", rev = "d6b81f5303b0a347726e1
 
 # In development.
 halo2 = { git = "https://github.com/zcash/halo2.git", rev = "236115917df9db45282fec24d1e1e36f275f71ab" }
-orchard = { git = "https://github.com/zcash/orchard.git", rev = "21b77d6ec56fc7e37aab9da68e88bc5dd754fe10" }
+orchard = { git = "https://github.com/nuttycom/orchard.git", rev = "aa0c0ecbecbb61b708434e4158464f8516a9e4b8" }
```
Comment (str4d) [inline Cargo.toml:27]: Change this now that https://github.com/zcash/orchard/pull/140 has merged.

File: zcash_primitives/src/transaction/components/orchard.rs [hunk 2]
```diff
@@ -185,7 +185,7 @@ pub fn write_v5_bundle<W: Write>(
 
         write_flags(&mut writer, &bundle.flags())?;
```
Comment (str4d) [inline zcash_primitives/src/transaction/components/orchard.rs:186]: Can this be replaced by the new `orchard` APIs? (And similarly for reading.)

**Review by str4d — APPROVED** [review]
utACK after `orchard` patch is updated.

File: zcash_primitives/src/transaction/components/orchard.rs (hunk 2 above)
Comment (nuttycom) [reply zcash_primitives/src/transaction/components/orchard.rs:194]: Serialization has not been moved to the `orchard` crate, just the construction of the txid and auth hashes.

Comment (nuttycom) [reply zcash_primitives/src/transaction/components/orchard.rs:194]: Oh, I guess though that `Flags::to_byte` has. Will do.

**Pull request merged**